client/src/app/containers/Game/index.tsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+
+import { Grid, IconButton } from '@material-ui/core';
+import RefreshIcon from '@material-ui/icons/Refresh';
+
+import { useAuth } from '../../../auth/useAuth';
+import ActionBox from './ActionBox';
+import CribHand from './CribHand';
+import { Game, Phase } from './models';
+import PlayerHand from './PlayerHand';
+import PlayingCard from './PlayingCard';
+import ScoreBoard from './ScoreBoard';
+import { useGame } from './useGame';
+
+const showCutCard = (phase: Phase) =>
+    !['Deal', 'BuildCrib', 'Cut'].includes(phase);
+
+const handsExist = (phase: Phase) =>
+    !['unknownPhase', 'Deal', 'DealingReady'].includes(phase);
+
+const handForPlayer = (
+    game: Game,
+    myID: string,
+    position: 'across' | 'right' | 'left',
+) => {
+    if (!handsExist(game.phase)) {
+        return [];
+    }
+    const numPlayers = game.teams.reduce(
+        (prev, team) => prev + team.players.length,
+        0,
+    );
+    const isFourPlayer = numPlayers === 4;
+    if (position === 'across') {
+        if (game.teams.length === 3) {
+            const secondPlayerID = game.teams.filter(
+                t => !t.players.some(p => p.id === myID),
+            )[1].players[0].id;
+            return game.hands[secondPlayerID] ?? [];
+        }
+        if (isFourPlayer) {
+            const partnerID = game.teams
+                .filter(t => t.players.some(p => p.id === myID))[0]
+                .players.filter(p => p.id !== myID)[0].id;
+            return game.hands[partnerID] ?? [];
+        }
+        const opponentID = game.teams.filter(
+            t => !t.players.some(p => p.id === myID),
+        )[0].players[0].id;
+        return game.hands[opponentID] ?? [];
+    }
+    if (position === 'right') {
+        if (isFourPlayer) {
+            const rightID = game.teams
+                .filter(t => t.players.some(p => p.id !== myID))[0]
+                .players.filter(p => p.id !== myID)[1].id;
+            return game.hands[rightID] ?? [];
+        }
+        // nothing!
+        return [];
+    }
+    if (position !== 'left' || !isFourPlayer) {
```

Comment:
:mag: is `position !== 'left'` possible now that we have an enum input?

Reply:
Yes, because technically `position` isn't typed as an enum (i.e. position does _not_ have an underlying value of 0, 1, etc. like an enum would), but is instead typed as a specific set of strings (called a union type). So the type could be expressed as "string, but there's a constaint: it has to be one of these three strings".

Reply:
When editing this file, for example, the editor will tell you which strings you can check for because it knows that `positions` only has a limited set of options.